PaulasWeatherApp/PaulasWeatherApp/Model/APIModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  APIModel.swift
+//  PaulasWeatherApp
+//
+//  Created by Stef, Paula on 7/26/22.
+//
+
+import Foundation
+
+class APIModel {
```

Comment:
A better name for this one is WeatherService -> because this class handles the service calls.